Sources/MapboxCoreNavigation/Directions+NavigationProviderRequest.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import MapboxDirections
+
+extension URLSessionDataTask: NavigationProviderRequest {
+    public var id: UInt64 {
```

Comment:
This extends a commonly used Swift standard library type with a commonly used property name (which happens to be part of the Identifiable protocol). Can we name this property something else less prone to conflicts?